packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal-content-wrapper.tsx

```diff
@@ -209,45 +209,40 @@ const JurisdictionModalContentWrapper = observer(({ openPasswordModal }: TJurisd
-                        <JurisdictionModalFootNote
-                            account_status={account_status}
-                            account_type={account_type.type}
-                            card_classname={`cfd-jurisdiction-card--${account_type.type}`}
-                            jurisdiction_selected_shortcode={jurisdiction_selected_shortcode}
-                            should_restrict_bvi_account_creation={should_restrict_bvi_account_creation}
-                            should_restrict_vanuatu_account_creation={should_restrict_vanuatu_account_creation}
-                        />
-                        <JurisdictionCheckBox
-                            is_checked={checked}
-                            onCheck={() => setChecked(!checked)}
-                            class_name={`cfd-jurisdiction-card--${account_type.type}__jurisdiction-checkbox`}
-                            jurisdiction_selected_shortcode={jurisdiction_selected_shortcode}
-                            should_restrict_bvi_account_creation={should_restrict_bvi_account_creation}
-                            should_restrict_vanuatu_account_creation={should_restrict_vanuatu_account_creation}
-                        />
-                    </div>
+            </div>
+            <div className={classNames('jurisdiction-modal__footer-content', `cfd-jurisdiction-card__footer-wrapper`)}>
+                <div className={`cfd-jurisdiction-card--${account_type.type}__footnotes-container`}>
+                    <JurisdictionModalFootNote
+                        account_status={account_status}
+                        account_type={account_type.type}
+                        card_classname={`cfd-jurisdiction-card--${account_type.type}`}
+                        jurisdiction_selected_shortcode={jurisdiction_selected_shortcode}
+                        should_restrict_bvi_account_creation={should_restrict_bvi_account_creation}
+                        should_restrict_vanuatu_account_creation={should_restrict_vanuatu_account_creation}
+                    />
+                    <JurisdictionCheckBox
+                        is_checked={checked}
+                        onCheck={() => setChecked(!checked)}
+                        class_name={`cfd-jurisdiction-card--${account_type.type}__jurisdiction-checkbox`}
+                        jurisdiction_selected_shortcode={jurisdiction_selected_shortcode}
+                        should_restrict_bvi_account_creation={should_restrict_bvi_account_creation}
+                        should_restrict_vanuatu_account_creation={should_restrict_vanuatu_account_creation}
+                    />
                 </div>
+                <Modal.Footer className='jurisdiction-modal__footer-button' has_separator>
+                    <Button
+                        disabled={isNextButtonDisabled()}
+                        primary
+                        style={{ width: is_mobile ? '100%' : 'unset' }}
```

Comment:
where are you getting `is_mobile` from as it is coming from ui store?

Reply:
oops